### ISSUE TITLE
can't get data submodule when master is cloned from https

### ISSUE THREAD

**Issue by Anton-V-K** [colobot/colobot#1534]
The config for data submodule explicitly uses git-protocol, which doesn't work smoothly when master is cloned through https-protocol.
I guess, the path to submodules should be relative or at least doesn't mention protocol explicitly.
The log:
```
D:\Temp>git clone https://github.com/colobot/colobot.git colobot
Cloning into 'colobot'...
remote: Enumerating objects: 43576, done.
remote: Counting objects: 100% (4264/4264), done.
remote: Compressing objects: 100% (1485/1485), done.
remote: Total 43576 (delta 3152), reused 3446 (delta 2731), pack-reused 39312
Receiving objects: 100% (43576/43576), 55.23 MiB | 1.19 MiB/s, done.
Resolving deltas: 100% (34519/34519), done.
D:\Temp>cd colobot
D:\Temp\colobot>git submodule update --init
Submodule 'data' (git://github.com/colobot/colobot-data.git) registered for path 'data'
Cloning into 'D:/Temp/colobot/data'...
fatal: unable to connect to github.com:
github.com[0: 140.82.121.3]: errno=Unknown error

fatal: clone of 'git://github.com/colobot/colobot-data.git' into submodule path 'D:/Temp/colobot/data' failed
Failed to clone 'data'. Retry scheduled
Cloning into 'D:/Temp/colobot/data'...
fatal: unable to connect to github.com:
github.com[0: 140.82.121.4]: errno=Unknown error

fatal: clone of 'git://github.com/colobot/colobot-data.git' into submodule path 'D:/Temp/colobot/data' failed
Failed to clone 'data' a second time, aborting
```
The content of `colobot\.git\config`:
```
[core]
	repositoryformatversion = 0
	filemode = false
	bare = false
	logallrefupdates = true
	symlinks = false
	ignorecase = true
[remote "origin"]
	url = https://github.com/colobot/colobot.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = origin
	merge = refs/heads/master
[submodule "data"]
	active = true
	url = git://github.com/colobot/colobot-data.git
```
Refer to [this answer](https://stackoverflow.com/a/60726312/536172) for a hint how to set up submodules in a relative way (see also [this question](https://stackoverflow.com/q/60712774/536172)).
Current content of `.gitmodules`:
```
[submodule "data"]
	path = data
	url = git://github.com/colobot/colobot-data.git
```
Proposed change in `.gitmodules` to fix the issue:
```
[submodule "data"]
	path = data
	url = ../colobot-data.git
```
Such fix supports correct initialization of submodules (for a freshly cloned repo):
```
D:\Temp\colobot>git submodule update --init
Submodule 'data' (https://github.com/colobot/colobot-data.git) registered for path 'data'
Cloning into 'D:/Temp/colobot/data'...
Submodule path 'data': checked out '0ac8197b7a8a005c714b7696d36c642cf0e81474'
```
Can be fixed in the context of #1306

**Comment by Anton-V-K** [comment]
`.gitmodule` should also explicitly specify branch (if it isn't `master`), because otherwise `data` won't be compatible with application sources (take a look into #1506 which is caused by branches mismatch).
So for `dev` branch `.gitmodule` will look like this:
```
[submodule "data"]
	path = data
	url = ../colobot-data.git
        branch = dev
```

**Comment by constXife** [comment]
I used this solution without modification files

https://stackoverflow.com/questions/4891527/git-protocol-blocked-by-company-how-can-i-get-around-that

**Comment by tomangelo2** [comment]
Fixed in 0.2.1 release.
